service/kas/kas.proto

```diff
@@ -31,52 +31,157 @@ message LegacyPublicKeyRequest {
   string algorithm = 1;
 }
 
+// Policy binding ensures cryptographic integrity between policy and wrapped key
+// Prevents policy tampering by binding the policy hash to the encrypted key
 message PolicyBinding {
+  // Cryptographic hashing algorithm used for policy binding
+  // Optional: ZTDF (when policy_binding is an object), NanoTDF (embedded in header)
+  // Value: Always "HS256" (HMAC-SHA256) - other algorithms not supported
+  // Example: "HS256"
   string algorithm = 1 [json_name = "alg"];
+  
+  // HMAC-SHA256 hash of the base64-encoded policy using the DEK as the secret key
+  // Required: ZTDF (when policy_binding is an object), NanoTDF (computed from header)
+  // Links the policy content to the wrapped DEK cryptographically via HMAC
+  // Computed as HMAC-SHA256(DEK, base64_policy) then hex-encoded and base64-encoded
```

Comment:
This is hex and base64 encoded for 4.2.2 TDFs; in 4.3.0+ it is just base64 encoded IIRC. Either way the example is misleading (unless it is base64 then hex encoded)